csrc/id_model/indexing.cpp

```diff
@@ -339,6 +372,27 @@ void TensorIndexer::buildLoopIndexMap() {
   }
 }
 
+bool TensorIndexer::shouldUseZeroIndex(const ValGroup& loop_group) const {
+  // All loops in this set are non-parallel, non-concretized broadcast
```

Comment:
So if all the axes are broadcast then we should use 0 index, or if the promoted loop has extent 1 (and is not partial). How does "non-parallel" affect this check?

Reply:
Ah, that comment should have been removed. I had some other code above this line. Thanks for catching it.